GDAS-validation/gdas_config/config.anal

```diff
@@ -0,0 +1,155 @@
+  fi
+
+  #   Assimilate HDOB
+  if [[ "${PDY}${cyc}" -ge "2020082412" && "${PDY}${cyc}" -lt "2020091612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020082412
+  fi
+
+  #   Assimilate Metop-C GNSSRO
+  if [[ "${PDY}${cyc}" -ge "2020091612" && "${PDY}${cyc}" -lt "2021031712" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020091612
+  fi
+
+  #   Assimilate DO-2 GeoOptics
+  if [[ "${PDY}${cyc}" -ge "2021031712" && "${PDY}${cyc}" -lt "2021091612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2021031712
+  fi
+
+  #   NOTE:
+  #   As of 2021110312, gfsv16_historical/global_convinfo.txt.2021110312 is
+  #   identical to ../global_convinfo.txt.  Thus, the logic below is not
+  #   needed at this time.
+  #   Assimilate COSMIC-2 GPS
+  #   if [[ "${PDY}${cyc}" -ge "2021110312" && "${PDY}${cyc}" -lt "YYYYMMDDHH" ]]; then
+  #   export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2021110312
+  #   fi
+
+  #   Turn off assmilation of OMPS during period of bad data
+  if [[ "${PDY}${cyc}" -ge "2020011600" && "${PDY}${cyc}" -lt "2020011806" ]]; then
+    export OZINFO=${FIXgsi}/gfsv16_historical/global_ozinfo.txt.2020011600
+  fi
+
+
+  #   Set satinfo for start of GFS v16 parallels
+  if [[ "${PDY}${cyc}" -ge "2019021900" && "${PDY}${cyc}" -lt "2019110706" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2019021900
+  fi
+
+  #   Turn on assimilation of Metop-C AMSUA and MHS
+  if [[ "${PDY}${cyc}" -ge "2019110706" && "${PDY}${cyc}" -lt "2020022012" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2019110706
+  fi
+
+  #   Turn off assimilation of Metop-A MHS
+  if [[ "${PDY}${cyc}" -ge "2020022012" && "${PDY}${cyc}" -lt "2021052118" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2020022012
+  fi
+
+  #   Turn off assimilation of S-NPP CrIS
+  if [[ "${PDY}${cyc}" -ge "2021052118" && "${PDY}${cyc}" -lt "2021092206" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2021052118
+  fi
+
+  #   Turn off assimilation of MetOp-A IASI
+  if [[ "${PDY}${cyc}" -ge "2021092206" && "${PDY}${cyc}" -lt "2021102612" ]]; then
+    export SATINFO=${FIXgsi}/gfsv16_historical/global_satinfo.txt.2021092206
+  fi
+
+  #   NOTE:
+  #   As of 2021110312, gfsv16_historical/global_satinfo.txt.2021110312 is
+  #   identical to ../global_satinfo.txt.  Thus, the logic below is not
+  #   needed at this time
+  #
+  #   Turn off assmilation of all Metop-A MHS
+  #   if [[ "${PDY}${cyc}" -ge "2021110312" && "${PDY}${cyc}" -lt "YYYYMMDDHH" ]]; then
+  #     export SATINFO=$FIXgsi/gfsv16_historical/global_satinfo.txt.2021110312
+  #   fi
+fi
+
+# UFO evaluation
+export USE_RADSTAT="NO" # This can be only used when bias correction is not-zero.
+export SETUP="miter=0,nhr_obsbin=6,${SETUP:-}"
+export JCAP_A=766
+export JCAP_ENKF=766
+export LONA=1536
+export LATA=768
+
```

Comment:
We can add lines here to reset convinfo and satinfo back to the intended fix versions.  
```
export CONVINFO=${FIXgsi}/global_convinfo.txt
export SATINFO=${FIXgsi}/global_satinfo.txt

```

Reply:
Lines added at 37fea66.